lib/matplotlib/colorbar.py

```diff
@@ -1437,7 +1437,7 @@ def make_axes(parents, location=None, orientation=None, fraction=0.15,
         if parent_anchor is not False:
             ax.set_anchor(parent_anchor)
 
-    cax = fig.add_axes(pbcb)
+    cax = fig.add_axes(pbcb, label="(colorbar)")
```

Comment:
```suggestion
    cax = fig.add_axes(pbcb, label="<colorbar>")
```

Looks more in line with

~~~
<anonymous AxesSubplot> (id: 0x...)
<anonymous AxesSubplot> (id: 0x...)
(colorbar) (id: 0x...)
(colorbar) (id: 0x...)
~~~

Reply:
done